libnmstate/dns.py

```diff
@@ -123,7 +123,9 @@ def gen_metadata(self, ifaces, route_state):
             if not iface_name:
                 raise NmstateValueError(
                     "Failed to find suitable interface for saving DNS "
-                    "name servers: %s" % server
+                    "name servers: %s, nmstate only support saving DNS to "
+                    "interface with static gateay or auto interface with "
```

Comment:
There is a typo in `gateway`. Other than that, it looks good. It is wrong in the commit message too.

Reply:
Fixed. Thanks.